pug/home.pug

```diff
@@ -57,10 +57,13 @@ block content
   pre #{''}
   
   div.border-danger.border.rounded.p-2
-    h5 Known Bugs:
+    h5 Bugs and Other Issues
     ul
-      li All pages - the page titles are not showing up in browser tabs
-      li All component pages - the component history is unavailable (null date error)
+      li Bugs and other errors with existing functionality should be reported at our 
+         |
+         a(href = "https://github.com/DUNE/dunedb/issues") Github Issues page
```

Comment:
Consider whether adding `target = "_blank"` would be better to avoid navigating away from the page if the user needs to report a bug.

Reply:
Will do.